app/views/manuals/_header.html.erb

```diff
@@ -1,5 +1,8 @@
-<header aria-labelledby="manual-title" class="<%= presented_manual.hmrc? ? 'hmrc' : nil %>">
-  <div class='primary'>
+<header
+  aria-labelledby="manual-title"
+  class="<%= presented_manual.hmrc? ? 'hmrc' : nil %> <%= local_assigns.has_key?(:new_navigation) ? 'new-navigation' : ''%>">
```

Comment:
Nitpick: Is there any need for the ternary operators here?  `'hmrc' if 'presented_manual.hmrc?'` is surely more readable and concise?

At the very least, can you please make the ternary operators consistent (one returns `nil` and the other a blank string!)?